tests/e2e/kepler_test.go

```diff
@@ -88,7 +93,8 @@ func TestNodeSelector(t *testing.T) {
 	err = f.AddResourceLabels("node", node, labels)
 	assert.NoError(t, err, "could not label node")
 	t.Cleanup(func() {
```

Comment:
@vprashar2929 thanks for the patch. Since you are at it,  could you also please  move `t.Cleanup` to `f.AddResourceLabels` itself.  There are other functions in the framework that automatically cleans up after itself so that the caller need not worry about cleaning up.